frontend_app_thread/src/container/Thread.jsx

```diff
@@ -392,16 +398,19 @@ export class Thread extends React.Component {
 
   render () {
     const { props, state } = this
+    const isPublication = state.content.content_namespace === CONTENT_NAMESPACE.PUBLICATION
+    const publicationColor = '#661F98'
```

Comment:
We have the same const at frontend/helper, I think it is a good idea to put this in frontend_lib and have the same const for both.

Reply:
done